docs/API_ACCESS.md

```diff
@@ -0,0 +1,62 @@
+The Search API is implemented with GraphQL. This document explains how to access the service to send requests.
+
+## Pre-requisite
+Red Hat Advanced Cluster Management installed on your OpenShift cluster.
+
+## Locate the Search API endpoint
+### From within the cluster
+If your application is deployed in the cluster, and it has the correct permissions, it can send requests to https://search-search-api.open-cluster-management.svc:4010/searchapi/graphql
+
+### From outside the cluster
+By default, the Search API is not exposed. Follow [this topic](https://docs.openshift.com/container-platform/4.9/networking/routes/secured-routes.html) to create a secure route to the service `search-search-api` in the `open-cluster-management` namespace.
+
+```
+# NOTE: this sample requires an insecure connection from the client.
+oc create route passthrough search-api --service=search-search-api -n open-cluster-management
+```
+
+The route should look like this:  `https://search-api-open-cluster-management.apps.[your-ocp-hostname]/searchapi/graphql`
```

Comment:
small clarification. The route that is created will be `https://search-api-open-cluster-management.apps.[your-ocp-hostname]/` . The Query that you will use will be: https://search-api-open-cluster-management.apps.[your-ocp-hostname]/searchapi/graphql`